playbooks/ci-full/templates/heat_stack.yml

```diff
@@ -0,0 +1,254 @@
+heat_template_version: 2013-05-23
+
+description: HOT template to deploy Ursula Tests env
+
+# Note: there should be a placeholder line after each "if-else" block to keep the right format of the template
+
+parameters:
+  image:
+    type: string
+    description: Name of image to use for servers
+    {% if ansible_env.IMAGE_ID -%}
+    default: {{ ansible_env.IMAGE_ID }}
+    {% else -%}
+    default: ubuntu-14.04
+    {% endif -%}
+    # placeholder line
+
+  flavor:
+    type: string
+    description: Flavor to use for servers
+    {% if ansible_env.testenv_flavor -%}
+    default: {{ ansible_env.testenv_flavor }}
+    {% else -%}
+    default: m1.large
+    {% endif -%}
+    # placeholder line
+
+  network:
+    type: string
+    description: Name of Neutron network into which servers get deployed
+    {% if ansible_env.testenv_network -%}
+    default: {{ ansible_env.testenv_network }}
+    {% else -%}
+    default: internal
+    {% endif -%}
+    # placeholder line
+
+  floating_ip_pool:
+    type: string
+    description: Name of floating ip pool to use
+    {% if ansible_env.testenv_floating_ip_pool -%}
+    default: {{ ansible_env.testenv_floating_ip_pool }}
+    {% else -%}
+    default: external
+    {% endif -%}
+    # placeholder line
```

Comment:
The default filter you suggested works well, I will switch to that for all these items. Thanks!